tests/Unit/stream_str_replace_Test.php

```diff
@@ -1,39 +1,39 @@
 <?php
-
-use function WordPress\Streams\stream_str_replace;
-
-it( "Replace a string in a stream", function () {
-	$source = fopen( 'php://memory', 'w+' );
-	fwrite( $source, "Hello, world, and all the other places too!" );
-	rewind( $source );
-
-	$generator = stream_str_replace( $source, "and all ", "" );
-	$chunks    = iterator_to_array( $generator );
-	$expected  = "Hello, world, the other places too!";
-
-	expect( implode( "", $chunks ) )->toBe( $expected );
-} );
-
-it( "Replace a URL in a longer string", function () {
-	$data = <<<EOT
-<!DOCTYPE html>
-<html>
-<head>
-	<title>Test</title>
-</head>
-<body>
-	<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-	<a href="https://example.com">Example</a>
-</body>
-EOT;
-
-	$source = fopen( 'php://memory', 'w+' );
-	fwrite( $source, $data );
-	rewind( $source );
-
-	$generator = stream_str_replace( $source, "https://example.com", "https://playground.internal" );
-	$chunks    = iterator_to_array( $generator );
-	$expected  = str_replace( "https://example.com", "https://playground.internal", $data );
-
-	expect( implode( "", $chunks ) )->toBe( $expected );
-} );
+//
+//use function WordPress\Streams\stream_str_replace;
```

Comment:
What's the issue with this test in PHP 7.4?

Reply:
That is a Pest test. And we can't have Pest dependencies since they require a higher version of PHPUnit then the one we want to have. Also, keeping both Pest and PHPUnit, were it possible, would needlessly complicate things.

This applies to all commented tests. I'll rewrite them. I'd just preffer to do in separate PRs.